CHANGELOG.rst

```diff
@@ -3,33 +3,48 @@
 0.20.0
 ------
 
-This release includes improvements to the UX on the UI, internal changes to the Knowledge Graph, a bugfix on the core service
-and support for running sessions in a separate namespace.
+Renku ``0.20.0`` tidies up the RenkuLab session start sequence, as well as an assortment of improvements and big fixes. 
 
-Features
-~~~~~~~~
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
 
-* **Renku core service**: add workflow export endpoint
-* **Renku CLI**: inform users about push in renku save command
-* **UI**: improve UX around sessions 💻 (unified start flow, better logs, error handling)
-* **UI & Gateway**: adopt the latest layout on login and logout pages 🎨
-* **Sessions**: support for running sessions in a separate namespace 
-* **Sessions**: caching jupyter servers
+**🌟 New Features**
 
-Bug fixes
-~~~~~~~~~~
+* *None this release*
+
+**✨ Improvements**
+
+* 🚀 **UI**: We tidied up the session launch sequence to have a unified appearance, regardless of where on RenkuLab you start your session from. Sessions also have better logs and error handling.
```

Comment:
I 🫶 emojis, but I feel they might be a bit confusing as the first element of bullet points because it's not immediately obvious what changes belong to UI and what to CLI.
What about inverting?
* 🚀 **UI** 
* **UI** 🚀

Reply:
Huh, I see it the other way around. With the emojis first, you get a columnar format of single-width-emoji - team name - description. If you put the emoji second, they aren't aligned, and I think it looks messier.